Lib/test/test_colorsys.py

```diff
@@ -96,5 +96,32 @@ def test_yiq_values(self):
             self.assertTripleEqual(yiq, colorsys.rgb_to_yiq(*rgb))
             self.assertTripleEqual(rgb, colorsys.yiq_to_rgb(*yiq))
 
+    def test_yuv_roundtrip(self):
+        for r in frange(0.0, 1.0, 0.2):
+            for g in frange(0.0, 1.0, 0.2):
+                for b in frange(0.0, 1.0, 0.2):
```

Comment:
Could you use [subtest](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests)?

Reply:
Added this on all of the other roundtrip tests as well.